src/commands/dev/index.js

```diff
@@ -172,7 +175,25 @@ async function startProxy(settings, addonUrls, configPath, projectDir) {
     configPath,
   })
 
+  await mkdirp(path.resolve(projectDir, '.netlify'))
+
+  const formSubmissionsFile = path.resolve(projectDir, '.netlify', 'form-submissions.csv')
+  const csvWriter = csvWriterPkg.createArrayCsvWriter({
+    alwaysQuote: true,
+    header: ['timestamp', 'path', 'body'],
+    path: formSubmissionsFile,
+    append: true,
+  })
+
   const server = http.createServer(function(req, res) {
+    if (req.method === 'POST') {
+      collectRequestBody(req, async body => {
```

Comment:
The `Content-Type` response header should be set to `text/csv`?

Reply:
most folks return only html from their form submission, so probably don't need to get too fancy.